build-data-capturing-gradle-samples/capture-test-pts-support/gradle-test-pts-support.gradle.kts

```diff
@@ -56,6 +57,19 @@ class Capture(val logger: Logger) {
         try {
             var engines = t.classpath.files.stream().filter { f -> f.name.endsWith(".jar") }
                 .filter { f -> supportedEngines.values.stream().anyMatch { e -> f.name.contains(e) } }
+                .filter { f ->
+                    if (f.name.contains("kotest-runner")) {
```

Comment:
I would extract all of this comparison logic to a function.

Reply:
Extracted the filter predicate to a property